components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/internal/APIManagerJWTAuthComponent.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.apimgt.rest.api.common.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.wso2.carbon.apimgt.impl.APIManagerConfigurationService;
+import org.osgi.service.component.ComponentContext;
+
+@Component(
+        name = "org.wso2.apimgt.rest.api.common",
+        immediate = true)
+public class APIManagerJWTAuthComponent {
```

Comment:
class comment

Reply:
we have to rename this class too I think.. This is in rest api common package. We have to use a name considering that.
APIMRestAPICommonComponent?